server/src/main/java/io/druid/server/http/HistoricalResource.java

```diff
@@ -52,4 +52,16 @@ public Response getLoadStatus()
   {
     return Response.ok(ImmutableMap.of("cacheInitialized", coordinator.isStarted())).build();
   }
+
+  @GET
+  @Path("/loadStatusCode")
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response getLoadStatusCode()
+  {
+    if (coordinator.isStarted()) {
+      return Response.ok(ImmutableMap.of()).build();
```

Comment:
If we only need the status code, how about `Response.ok().build()`?

Reply:
Changed